files/ru/learn/css/building_blocks/the_box_model/index.html

```diff
@@ -282,91 +284,75 @@ <h3 id="Padding">Padding</h3>
 
 <p><strong>Если вы измените значения padding в классе <code>.box</code> в примере ниже, вы можете увидеть что это изменяет положение текста относительно поля.</strong></p>
 
-<p><strong>Вы также можете изменить padding в классе <code>.container,</code> который создаёт пространство между контейнером и боксом. Padding может быть изменён для любого элемента, и создать пространство между этой и любой другой границей внутри элемента.</strong></p>
+<p><strong>Вы также можете изменить padding в классе <code>.container,</code> который создаёт пространство между контейнером и блоком. Padding может быть изменён для любого элемента, и создать пространство между этой и любой другой рамкой внутри элемента.</strong></p>
```

Comment:
```suggestion
<p><strong>Вы также можете изменить padding для класса <code>.container</code>, который задаёт отступ между контейнером и блоком. Внутренний отступ может быть изменён для любого элемента и создаст пространство между его рамкой и содержимым.</strong></p>
```
Предлагаю так. Хотя всё равно как-то кривовато звучит, можно покрутить ещё)